core/src/main/java/zipkin2/reporter/AsyncReporter.java

```diff
@@ -282,7 +282,7 @@ void flush(BufferNextMessage<S> bundler) {
 
       try {
         sender.sendSpans(nextMessage).execute();
-      } catch (IOException | RuntimeException | Error t) {
```

Comment:
I'm wondering why we had this level of specificity before. What types of exceptions did we not want to catch and is it okay to catch them now?

Reply:
The specificity hasn't changed - the previous signature caught all unchecked exceptions, and the only possible checked exception `IOException`. But I find this to be less readable than just going with `Throwable` which is semantically perfect (any thrown exception) and always less code than explicitly listing checked exceptions.

Sorry for adding this cleanup without pointing it out it's admittedly unrelated to the issue.